java/images/centos-java11/README.md

```diff
@@ -118,6 +118,12 @@ Example:
 export MAVEN_OPTS="$(run-java.sh options --memory)"
 mvn clean install
 ```
+
+
+undefined
```

Comment:
`undefined` may be clerical error.

Reply:
This file was generated by `fish-pepper`.